test/csuite/schema_abort/main.c

```diff
@@ -605,14 +606,18 @@ thread_ckpt_run(void *arg)
              */
             if (!stable_set) {
                 __wt_epoch(NULL, &now);
-                if (WT_TIMEDIFF_SEC(now, start) >= 1)
-                    printf("CKPT: !stable_set time %" PRIu64 "\n", WT_TIMEDIFF_SEC(now, start));
-                if (WT_TIMEDIFF_SEC(now, start) > MAX_STARTUP) {
+                diff_sec = WT_TIMEDIFF_SEC(now, start);
+                if (diff_sec >= 1)
```

Comment:
This will always be true once it has reached one second as now - start will increase over time.

Reply:
Thanks, @etienneptl . I plan to print this on every 100th iteration.

Reply:
Fixed in 2e419208e0e13ddac1d9737a66977b1a612c49f5